network_observability/observing-network-traffic.adoc

```diff
@@ -16,19 +16,17 @@ include::modules/network-observability-pktdrop-overview.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
 .Additional resources
-* xref:../network_observability/observing-network-traffic.adoc#network-observability-packet-drops_nw-observe-network-traffic[Working with packet drops]
+* For more information about configuring packet drops in the `FlowCollector`, see xref:../network_observability/observing-network-traffic.adoc#network-observability-packet-drops_nw-observe-network-traffic[Working with packet drops].
```

Comment:
Same comment about preferring these to be single xrefs. If you do keep it as a sentence, the below one is missing a period